frontend/src/pages/analytics/segments/UserSegmentDetails.tsx

```diff
@@ -0,0 +1,616 @@
+                          </Box>
+                          <Box margin={{ top: 'xxs' }}>
+                            {getRefreshSchedule(segment)}
+                          </Box>
+                        </div>
+                        <div>
+                          <Box variant="awsui-key-label">
+                            {t('analytics:segment.details.refreshExpiration')}
+                          </Box>
+                          <Box margin={{ top: 'xxs' }}>
+                            {getAutoRefreshExpiration(segment)}
+                          </Box>
+                        </div>
+                        <div>
+                          <Box variant="awsui-key-label">
+                            {t('analytics:segment.details.createdBy')}
+                          </Box>
+                          <Box margin={{ top: 'xxs' }}>
+                            {getCreatedByInfo(segment)}
+                          </Box>
+                        </div>
+                        <div>
+                          <Box variant="awsui-key-label">
+                            {t('analytics:segment.details.updatedBy')}
+                          </Box>
+                          <Box margin={{ top: 'xxs' }}>
+                            {getUpdatedByInfo(segment)}
+                          </Box>
+                        </div>
+                      </ColumnLayout>
+                    </Box>
+                    <Box margin={{ vertical: 'l' }}>
+                      {constructSegmentTrendChart()}
+                    </Box>
+                    <Table
+                      loading={isJobsLoading}
+                      header={
+                        <Header
+                          variant="h2"
+                          description={t(
+                            'analytics:segment.details.segmentHistoryDesc'
+                          )}
+                        >
+                          {t('analytics:segment.details.segmentHistory')}
+                        </Header>
+                      }
+                      variant="borderless"
+                      columnDefinitions={[
+                        {
+                          id: 'date',
+                          header: t('analytics:segment.details.date'),
+                          cell: (e: SegmentJobStatusItem) => e.date,
+                        },
+                        {
+                          id: 'status',
+                          header: t('analytics:segment.details.jobStatus'),
+                          cell: (e: SegmentJobStatusItem) =>
+                            constructJobStatusCell(e.jobStatus),
+                        },
+                        {
+                          id: 'startTime',
+                          header: t('analytics:segment.details.startTime'),
+                          cell: (e: SegmentJobStatusItem) =>
+                            new Date(e.jobStartTime).toLocaleString(),
```

Comment:
use `moment(e.jobStartTime).format(TIME_FORMAT)` for date ?

Reply:
done